.gitignore

```diff
@@ -53,5 +53,10 @@ build-iPhoneSimulator/
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
 
+node_modules/
+package-lock.json
+package.json
```

Comment:
I think package.json should not be gitignored?

Reply:
Or is it ignored because we should not use node modules? ;-)

Reply:
it is ignored because we don't use them now but I want to be able to load tools like eslint locally. As soon as we really need node_modules, we can remove the ignore. until then, I prefer not to put my local dev-tools in the package.json of the repository